### PR TITLE
Add doc summary for tf.strings.upper and tf.strings.lower.

### PR DESCRIPTION
**Pull request by pbaranay**
I think this is the right way of fixing #35647.

Recompiling and previewing the documentation is taking quite a while (following the instructions from [here](https://github.com/tensorflow/tensorflow/blob/f40a875355557483aeae60ffcf757fc9626c752b/tensorflow/g3doc/README.txt)), so I figured I'll just open the pull request now, and check out the preview tomorrow.

### CODE REVIEW

**Comment by pbaranay** [comment]
Sorry for the force-push, I noticed a typo in the commit description.

File: tensorflow/core/api_def/base_api/api_def_StringUpper.pbtxt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 op {
   graph_op_name: "StringUpper"
+  description: "Converts each string in the input Tensor to lowercase."
```
Comment (alextp) [inline tensorflow/core/api_def/base_api/api_def_StringUpper.pbtxt]: This should read uppercase I think

Comment (pbaranay) [reply tensorflow/core/api_def/base_api/api_def_StringUpper.pbtxt]: Thanks, fixed now.

**Review by alextp — APPROVED** [review]

**Comment by pbaranay** [comment]
It looks like the Bazel builds failed with "exited with error code 1" but no details besides that. Is there anything I can/should do to address this?

**Comment by mihaimaruseac** [comment]
There is an issue on the windows builds but we can ignore them at the moment.

This should soon get imported internally and merged from there, I don't think there's anything you need to do @pbaranay

**Pull request merged**